geemdjem0819/Assets/scripts/DragManager.cs

```diff
@@ -0,0 +1,67 @@
+﻿using UnityEngine;
+
+public class DragManager : MonoBehaviour
+{
+    public float catchingDistance = 3f;
+    public GameObject draggingObject;
+    private bool isDragging { get; set; }
```

Comment:
De speler of de manager is aan het dragen?

Reply:
We hebben geen object wat de speler voor moet stellen in de scene toch?

Reply:
"Mouse is dragging dan"?